src/vcu/src/can_handles.c

```diff
@@ -293,6 +293,17 @@ void handleDashRequest(Frame *msg) {
   control_settings.active_aero_enabled = unpacked_msg.active_aero_enabled;
 }
 
+void handle_DashRequestLC(Frame *msg) {
+  can0_DashRequestLC_T unpacked_msg;
+
+  unpack_can0_DashRequestLC(msg, &unpacked_msg);
+
+  if (unpacked_msg.launch_ctrl_slip_ratio != 65535) {
```

Comment:
What is 65535? Should this really go in the Dash Request handler?

Reply:
-1 is the convention for "hey vcu use your default value". Should've been -1 cast to the correct type.